alinka/widget/containers/main_body/content/settings/settings_tabs/team_member_tab/team_member_table_group.py

```diff
@@ -18,6 +18,7 @@ class TeamMemberTableModel(QAbstractTableModel):
     def __init__(self):
         self.insert_row = None
         self.columns = TeamMember.__table__.columns.keys()
+        self.header_labels = ["id", "Imię i nazwisko", "Specjalizacja"]
```

Comment:
In #100 it's `funkcja`, not `Specjalizacja`.

As general note - we also should research better place to put Polish-localized names, if we plan to re-use pattern of extended `QAbstractTableModel` (and I know that you already prepared pull request with another implementation of that class).

Reply:
Agree. I'm not very proud of this solution. It's rather quick fix before release.